latch_cli/services/init/example_docker/__init__.py

```diff
@@ -0,0 +1,64 @@
+"""
+Minimal template workflow to show how to run workflows that use docker in Latch
+
+For a more comprehensive template, see the assemble_and_sort workflow
+For examples on how to use docker in Latch, see https://docs.latch.bio/examples/workflows_examples.html
+"""
+
+from wf.task import blastp_task
+
+from latch import workflow
+from latch.resources.launch_plan import LaunchPlan
+from latch.types import (
+    LatchAuthor,
+    LatchFile,
+    LatchMetadata,
+    LatchOutputDir,
+    LatchParameter,
+)
+
+metadata = LatchMetadata(
+    display_name="biocontainers/blastp",
+    author=LatchAuthor(
+        name="biocontainers",
+    ),
+    parameters={
+        "query_file": LatchParameter(
+            display_name="Input FASTA File",
```

Comment:
add some rules to enforce `.fastq` extensions here so that people can see how to use them